mods/cnc/weapons/smallcaliber.yaml

```diff
@@ -114,85 +89,59 @@ M16:
 			Wood: 25
 			Light: 30
 			Heavy: 10
-		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
 	Warhead@2Eff: CreateEffect
 		Explosions: piff
 
+Pistol:
+	Inherits: ^LightMG
```

Comment:
I know, but due to the tiny default damage of 1, this gets rounded down to 0 for anything not infantry, so it shouldn't matter and saves some yaml lines.